src/vm/jitinterface.cpp

```diff
@@ -9982,6 +9982,25 @@ void* CEEInfo::getPInvokeUnmanagedTarget(CORINFO_METHOD_HANDLE method,
 
     if (pMD->NDirectTargetIsImportThunk())
     {
+        if (pMD->IsEarlyBound()
+#ifdef FEATURE_MULTICOREJIT
+            // Bug 126723: Calling ClassInit in multicore JIT background thread, return NULL
```

Comment:
Is this bug still relevant?

Reply:
The problem that this is talking is still relevant and it is actually worse in CoreCLR since we have tiered JIT.

`getPInvokeUnmanagedTarget` does not seem to be ever called by RuyJIT. We may want to change the implementation of `CEEInfo::getPInvokeUnmanagedTarget` to return null / assert in CoreCLR. (The JIT folks are keeping the JIT/EE interface same between CoreCLR and desktop, so we want to keep the method on the JIT/EE interface.)

Reply:
Switched to null/assert